src/test/regress/expected/multi_insert_select.out

```diff
@@ -1475,3 +1475,238 @@ DEBUG:  ProcessUtility
 CREATE VIEW test_view AS SELECT * FROM raw_events_first;
 INSERT INTO raw_events_second SELECT * FROM test_view;
 ERROR:  cannot plan queries that include both regular and partitioned relations
+-- we need this in our next test
+truncate raw_events_first;
+SET client_min_messages TO DEBUG4;
```

Comment:
Do we really have to run all this with debug4? It's a) hard to see and verify the important output b) is likely to change with other tests being added etc.